src/metiq.py

```diff
@@ -1166,6 +1166,18 @@ def __call__(self, parser, namespace, values, option_string=None):
         default=0,
         help="Filter latency outliers by calculating z-scores and filter above this value. Typical value is 3.`",
     )
+    parser.add_argument(
+        "--no-parse-clean",
+        action="store_true",
+        dest="no_parse_clean",
+        help="Do not try to clean single frame parsing errors.",
+    )
+    parser.add_argument(
+        "--filter-single-frames",
+        action="store_true",
```

Comment:
If you want to add a bool CLI option (e.g. called "foo-bar"), add 2x options: "--foo-bar" and "--nofoo-bar". Have both arguments write to a variable called "foo_bar". 

An example:
```
    parser.add_argument(
        "--json",
        dest="json_output",
        action="store_true",
        default=default_values["json_output"],
        help="Diff output in scriptable mode (1 JSON line)%s"
        % (" [default]" if default_values["json_output"] else ""),
    )
    parser.add_argument(
        "--no-json",
        dest="json_output",
        action="store_false",
        help="Diff output in non-scriptable mode%s"
        % (" [default]" if not default_values["json_output"] else ""),
    )
```